tests/appsec/iast/sink/test_command_injection.py

```diff
@@ -26,20 +26,23 @@ class TestCommandInjection:
     def setup_insecure(self):
         self.sink_fixture.setup_insecure()
 
+    @missing_feature(context.library < "dotnet@2.28.0", reason="Not implemented yet")
```

Comment:
Useless, as it's already declared in manifest file

Reply:
Cool. Thanks!